app/templates/components/high-charts.hbs

```diff
@@ -1 +1,2 @@
-{{yield}}
+<div class="chart-container"></div>
```

Comment:
Thanks for the review - yeah reason being that if the yielded content was wrapped as you suggest, that content would be replaced by the chart when it is created.